operator/api/v1alpha1/common/phases.go

```diff
@@ -33,6 +37,37 @@ func (p KeptnPhaseType) IsPostTask() bool {
 	return strings.Contains(p.ShortName, "PostDeployTasks")
 }
 
+func GetShortPhaseName(phase string) string {
+	var phases = []KeptnPhaseType{
+		PhaseWorkloadPreDeployment,
+		PhaseWorkloadPostDeployment,
+		PhaseWorkloadPreEvaluation,
+		PhaseWorkloadPostEvaluation,
+		PhaseWorkloadDeployment,
+		PhaseAppPreDeployment,
+		PhaseAppPostDeployment,
+		PhaseAppPreEvaluation,
+		PhaseAppPostEvaluation,
+		PhaseAppDeployment,
+		PhaseCompleted,
+		PhaseCancelled,
+	}
```

Comment:
Should we create this globally so we don't need to create the slice at each function invocation?

Reply:
in the future if we use it as part of another test/production code we can make it global, but for now I do not see a reason for that

Reply:
Sorry, I explain myself wrongly. It is not exposing this slice outside but rather avoiding allocating the memory over and over at each invocation of `GetShortPhaseName`